modular_skyrat/modules/opposing_force/code/mind.dm

```diff
@@ -25,8 +25,5 @@
 		to_chat(src, span_warning("You are OPFOR banned!"))
 		return
 
-	if(!mind.opposing_force)
-		var/datum/opposing_force/opposing_force = new(mind)
-		mind.opposing_force = opposing_force
-		SSopposing_force.new_opfor(opposing_force)
+	SSopposing_force.give_opfor_datum(mind)
```

Comment:
Didn't you already give them the OPFOR datum? In this case I don't think it would be necessary.

A side-effect of this affecting everyone is that it would spam us in the Show Antagonists panel.

Reply:
I don't see much of a better way to do this

Reply:
Did this in a better way that prevents spam and redundancy